docs/DeveloperGuide.md

```diff
@@ -160,26 +160,31 @@ How the parsing works:
 
 **API** : [`Model.java`](https://github.com/AY2122S1-CS2103T-T15-1/tp/tree/master/src/main/java/gomedic/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
-
+![ModelClassDiagram](images/ModelClassDiagram.png)
+![EntryModelClassDiagram](images/EntryModelClassDiagram.png)
```

Comment:
You might want to check this diagram again, seems there are some double arrows at the end

Reply:
do u mean inside the second diagram? i was thinking coz there are two arrows pointing at patient/doctor etc in the first diagram so the same amount of arrows would be pointing at them in the second diagram

Reply:
@simonjulianl

Reply:
I am not sure but using this class diagram is a bit messy, maybe using the normal class diagram like this is neater 
![Uploading image.png…]()

Reply:
Also matching the number of arrows hem .. i think its very unnecessary haha since if there are more diagrams pointing to doctor, then you need to draw tons of arrow lol, I think the arrow in to each model is not necessary

Reply:
yep ok

Reply:
yea this looks much better ill try and change it to this format instead.

Reply:
actl isnt the label with the C and A and whatnot wrong according to what we are taught tho

Reply:
like the notations on this class diagram is wrong

Reply:
nvm i found a change lol

Reply:
You can remove them by removing the skin